src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,95 @@
 package racingcar;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+class Car {
+	String name;
+	int step;
+	String show;
+
+	public Car(String n) {
+		name = n;
+		step = 0;
+		show = "";
+	}
+
+	public void go() {
+		int randomNumber = Randoms.pickNumberInRange(1, 9);
```

Comment:
2주차 피드백에 `값을 하드 코딩하지 않는다`는 내용이 있었습니다.

 1, 9와 같은 매직넘버는 상수화하면 좋을 것 같습니다 !

```suggestion
		int randomNumber = Randoms.pickNumberInRange(MIN_INCLUSIVE, MAX_INCLUSIVE);
```